alf-api/src/main/java/io/axway/alf/formatter/JsonWriter.java

```diff
@@ -56,10 +66,14 @@ private void write(@Nullable Object object) {
         }
     }
 
-    private void write(Consumer<Arguments> argsConsumer) {
-        JsonWriter jsonWriter = new JsonWriter(m_sb);
-        argsConsumer.accept(jsonWriter);
-        jsonWriter.end();
+    private void write(@Nullable Consumer<Arguments> argsConsumer) {
```

Comment:
support of null argsConsumer ?
could be inlined directly in the corresponding add method ?